.ci.yaml

```diff
@@ -3185,8 +3185,7 @@ targets:
     properties:
       dependencies: >-
         [
-          {"dependency": "xcode", "version": "14c18"},
-          {"dependency": "gems", "version": "v3.3.14"}
+          {"dependency": "ruby", "version": "ruby_3.1-pod_1.13"}
```

Comment:
we don't need xcode? or is it already included via the top level `mac.properties.$flutter/osx_sdk` field?

Reply:
I just copied dependencies from another target 🤷

Reply:
if it passes with less deps, i guess that's a good thing :)